client/my-sites/checkout/cart/style.scss

```diff
@@ -1,419 +0,0 @@
-.is-section-checkout .layout__secondary {
-	border-color: var( --color-border-subtle );
-}
-
-.cart__header {
-	border-top: 1px solid var( --color-border-subtle );
-
-	&.is-compact {
-		padding: 11px 15px;
-	}
-}
-
-.cart-body {
```

Comment:
I'm seeing some of those selectors in our e2e tests (`.cart-body` for example). Have we run all tests? Should we further clean some of them up?

Reply:
IIRC the code in `test/e2e/lib` is no longer in use by anything since we use the `calypso-e2e` package now. I'm not sure if it's worth editing those files here if there's a plan to remove them.

Reply:
See the comment here: https://github.com/Automattic/wp-calypso/pull/58467#issuecomment-978384765